.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: csharp
+solution: Mono.Cecil.sln
+install:
+  - nuget restore Mono.Cecil.sln
```

Comment:
Then it's not (but it doesn't fail either, and will be required if one day Cecil decides that some NuGet is worth using).
Travis calls it by default in the install step, so once you override the step (to install NUnit runner), you need to call it explicitely.
TL;DR it can be removed